packages/@coorpacademy-components/src/atom/chip/index.js

```diff
@@ -0,0 +1,77 @@
+import React, {useMemo, useState, useCallback} from 'react';
+import PropTypes from 'prop-types';
+import classnames from 'classnames';
+
+import get from 'lodash/fp/get';
+import Color from 'colorjs.io';
+import {FontAwesomeIcon} from '@fortawesome/react-fontawesome';
+import Provider from '../provider';
+import style from './style.css';
+
+const DELTA_LUMINOSITY = 0.08;
+const DEFAULT_BACKGROUND_COLOR = '#0061FF';
+const ICON_SIZE = '12px';
+
+const Chip = (props, context) => {
+  const {text, selected = false, onClick, backgroundColor = DEFAULT_BACKGROUND_COLOR} = props;
```

Comment:
I wonder if we should just rename props `backgroundColor` to `color` in that context
```suggestion
  const {text, selected = false, onClick, color = DEFAULT_BACKGROUND_COLOR} = props;
```

Reply:
in case we decide to add other color props in the future (text color, iconColor etc ...) it could be confusing to use color, I think that backgroundColor has 0% of confusion

Reply:
👌